testing/spectest/shared/bellatrix/ssz_static/ssz_static.go

```diff
@@ -0,0 +1,188 @@
+package ssz_static
+
+import (
+	"context"
+	"encoding/hex"
+	"errors"
+	"path"
+	"testing"
+
+	fssz "github.com/ferranbt/fastssz"
+	"github.com/golang/snappy"
+	v3 "github.com/prysmaticlabs/prysm/beacon-chain/state/v3"
+	ethpb "github.com/prysmaticlabs/prysm/proto/prysm/v1alpha1"
+	"github.com/prysmaticlabs/prysm/testing/require"
+	"github.com/prysmaticlabs/prysm/testing/spectest/utils"
+	"github.com/prysmaticlabs/prysm/testing/util"
+)
+
+// SSZRoots --
+type SSZRoots struct {
+	Root        string `json:"root"`
+	SigningRoot string `json:"signing_root"`
+}
+
+// RunSSZStaticTests executes "ssz_static" tests.
+func RunSSZStaticTests(t *testing.T, config string) {
+	require.NoError(t, utils.SetConfig(t, config))
+
+	testFolders, _ := utils.TestFolders(t, config, "bellatrix", "ssz_static")
+	for _, folder := range testFolders {
+		innerPath := path.Join("ssz_static", folder.Name(), "ssz_random")
+		innerTestFolders, innerTestsFolderPath := utils.TestFolders(t, config, "bellatrix", innerPath)
+		for _, innerFolder := range innerTestFolders {
+			t.Run(path.Join(folder.Name(), innerFolder.Name()), func(t *testing.T) {
```

Comment:
Please see PR #10003 and add the same changes to support the different ssz static modes.

Reply:
Done. Thanks for the feedback